core/opendaq/signal/include/opendaq/signal.h

```diff
@@ -128,6 +128,12 @@ DECLARE_OPENDAQ_INTERFACE(ISignal, IComponent)
      * Method returns OPENDAQ_IGNORED if that is the case.
      */
     virtual ErrCode INTERFACE_FUNC setStreamed(Bool streamed) = 0;
+
+    /*!
+     * @brief Gets the signal last value
+     * @param[out] value The signal last value
```

Comment:
Comments should explain how to use the value. It's unclear on why this is a base object, and how it should be interpreted (eg. how to cast it to the appropriate type and how to know the appropriate type).

Limitations (only floating point and integer signals) should be documented.